pkg/agent/openflow/pipeline.go

```diff
@@ -607,9 +638,32 @@ func (c *client) connectionTrackFlows(category cookie.Category) []binding.Flow {
 				Done(),
 		)
 	}
+
+	// Add reject response packet bypass flow.
+	flows = append(flows, c.conntrackBypassRejectFlow())
+
 	return flows
 }
 
+// conntrackBypassRejectFlow generates a flow which is used to bypass the reject
+// response packet sent by the controller to avoid unexpected packet drop.
+func (c *client) conntrackBypassRejectFlow() binding.Flow {
+	connectionTrackStateTable := c.pipeline[conntrackStateTable]
+	flowBuilder := connectionTrackStateTable.BuildFlow(priorityLow+1).
+		MatchProtocol(binding.ProtocolIP).
+		MatchRegRange(int(marksReg), CustomReasonReject, CustomReasonMarkRange).
+		Cookie(c.cookieAllocator.Request(cookie.Default).Raw())
+	if c.enableProxy {
```

Comment:
Since we already know this is a reply packet, does it need to go to sessionAffinityTable and serviceLBTable which are for new connections? And it could just be high priority?

Reply:
Changed.